jena-geosparql/pom.xml

```diff
@@ -105,7 +105,7 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-surefire-plugin</artifactId>
             <configuration>
-              <skip>true</skip>
+              <skip>false</skip>
```

Comment:
If we're no longer skipping we should remove this configuration entirely as otherwise it may prevent end users from opting in to skip tests

Reply:
I've removed the `surefire` plugin configuration block.